neuralmonkey/readers/plain_text_reader.py

```diff
@@ -14,6 +14,11 @@
 
 csv.field_size_limit(sys.maxsize)
 
+_ALNUM_CHARSET = set(
```

Comment:
remove the leading `_` (especially when you importing this stuff from another module)